packages/components/src/__future__/Tag/Tag/_docs/MigrationGuide.mdx

```diff
@@ -0,0 +1,168 @@
+
+Note: because the icons convey meaning, they should have an accessible label that provides context to the tag. A contextual `aria-label`and role of `img` addresses one of the [accessibility issues](https://github.com/cultureamp/kaizen-discourse/issues/102) with the legacy Tag.
+
+#### Before
+
+<Canvas of={LegacyTagStories.Validation} />
+
+#### After
+
+<Canvas of={TagMigrationStories.ValidationMigration} sourceState="shown"/>
+
+
+Note: When using the `icon` prop, the icon's color will be determined by `color` value that has been set. This is to ensure that and accessible contrast ratio is achieved.
+
+Any Icons added as a child of the `Tag` will need to be manually set to ensure the right contrast ratio.
+
+### Sentiment
+
+While migration of any `Tag` using, `sentimentPositive`, `sentimentNeutral` or `sentimentNegative` is relatively straight forward using the `color` prop, `sentimentNone` does not align to any Kaizen's [semantic colors](https://cultureamp.atlassian.net/wiki/spaces/DesignSystem/pages/3064857902/Color).
+
+#### Before
+
+<Canvas of={LegacyTagStories.Sentiments} />
+
+#### After
+
+<Canvas of={TagMigrationStories.SentimentsMigration} sourceState="shown"/>
+
+If required, you could use `classNameOverride` to implement the same UI. It is important that any override of the default styles still meets minimum contrast ratios to ensure the component is accessible.
+
+<Canvas of={TagMigrationStories.SentimentNone} />
+
+
+### Dismissible
+
+The `dismissible` prop is no longer available in the base `Tag` component. We have split the dismissible functionality into the [RemovableTag](/docs/components-tag-future-removabletag--docs).
+
+This increases the hit box for the dismiss button and makes the `ariaLabel` in the `removeButtonProps` a required prop. This offers a more accessible experience for user and address one of the standing [accessibility issues](https://github.com/cultureamp/kaizen-discourse/issues/101) with the legacy `Tag`.
+
+#### Before
+
+<Canvas of={LegacyTagStories.Dismissible} />
+
+#### After
+
+<Canvas of={TagMigrationStories.DismissibleMigration} sourceState="shown"/>
+
+Remember that when handling dismiss (or any destructive action), a users focus should not be lost on the page.
+
+
+### Avatars
+
+Avatars are no longer provided directly to the `Tag` as a prop. Instead they will now me passed in as children.
+
+#### Before
+
+<Canvas of={LegacyTagStories.ProfileWithAvatarProps} />
+
+#### After
+
+<Canvas of={TagMigrationStories.AvatarMigration} sourceState="shown"/>
+
+
+Note: The use of `ps-4` (or `padding-inline-start`) in the `classNameOverride` is to override the default spacing at start of the `Tag`. This will ensure the Avatar can contour the border radius of the `Tag`.
+
+#### Dismissible Avatars
+
+Following a similar pattern to the above, you can use the `RemovableTag` with an `Avatar` and plain text as its children to achieve a dismissible version.
```

Comment:
```suggestion
Following a similar pattern to the above, you can use the `RemovableTag` with an `Avatar` and plain text as its `children` to achieve a dismissible version.
```

I don't know whether `plain text` is required to be mentioned. Technically it can accept a ReactNode right?